src/main/java/seedu/duke/Duke.java

```diff
@@ -2,10 +2,14 @@
 
 import java.time.LocalDateTime;
 import java.time.Month;
-//import java.util.ArrayList;
+import java.util.ArrayList;
 import java.util.Scanner;
 
+import static seedu.duke.ActivityList.INITIALISE;
+
 public class Duke {
+    public static ArrayList<Activity> activities = new ArrayList<>();
+
```

Comment:
Hi i think it would be better to merge the Activity class with the ActivityList class that i created, ie change the relevant Integer type in the activityList class to Activity class

Reply:
noted